assets/scripts/main.js

```diff
@@ -46,4 +48,57 @@
       });
     }
   });
+
+  // Single product sale label DOM element.
+  const $singleProductSaleLabel = $('.single-product-summary .onsale');
+
+  /**
+   * Hides sale label if percentage value is lower than 10%.
+   */
+  $('.onsale').each(function hideSaleLabel() {
+    if (parseInt($(this).text().replace(/^\D+/g, ''), 10) < shop_standards_settings.saleMinAmount) {
+      $(this).hide();
+    }
+  });
+
+  /**
+   * Updates sale percentage value.
+   *
+   * @param {Integer} percentage
+   *   The product/variation sale percentage.
+   */
+  function updateSaleLabel(percentage) {
+    if (percentage >= shop_standards_settings.saleMinAmount) {
+      $singleProductSaleLabel.text(`-${percentage}%`);
+      $singleProductSaleLabel.show();
+    } else {
+      $singleProductSaleLabel.hide();
+    }
+  }
+
+  $('.single_variation_wrap')
+    .on('show_variation', (event, variation) => {
+      if (variation.display_price < variation.display_regular_price) {
+        // eslint-disable-next-line max-len
```

Comment:
Can we move this onto the line itself and use `// eslint-disable-line max-len`? Would be a bit neater and wouldn't interrupt the reading flow.